packages/llvm20_build.rb

```diff
@@ -32,6 +32,10 @@ class Llvm20_build < Package
   depends_on 'xzutils' # R
   depends_on 'zlib' # R
   depends_on 'zstd' # R
+  depends_on 'llvm20_dev' # R
```

Comment:
Should these be here?

Reply:
Should be fixed.

Reply:
They're still there, unless i'm misunderstanding something?